presto-main/src/main/java/com/facebook/presto/cost/ComposableStatsCalculator.java

```diff
@@ -82,6 +82,10 @@ public PlanNodeStatsEstimate calculateStats(PlanNode node, StatsProvider sourceS
 
     public interface Rule<T extends PlanNode>
     {
+        /*
```

Comment:
Use the /**
                     *
                     */
Style of comment.

This should also be on the interface not the method.

Reply:
Okay. Done.